src/tools/kdb/ls.cpp

```diff
@@ -21,12 +21,9 @@ LsCommand::LsCommand () : kdb (root)
 {
 }
 
-int LsCommand::execute (Cmdline const & cl)
+int LsCommand::execute (const Cmdline & cl)
```

Comment:
is there any difference between Cmdline const and const Cmdline? I used const before the class type all the time, also in kdb complete. Should i change it too in there?

and what about something like `const Key key` (without &), should it also be `Key const key`?